src/Home/ProximityTracingActivationStatus.tsx

```diff
@@ -37,7 +38,7 @@ export const ProximityTracingActivationStatus: FunctionComponent = () => {
 
   const handleOnPressFix = () => {
     exposureNotifications.request()
-    if (isUnauthorized) {
+    if (isUnauthorized && isPlatformiOS()) {
```

Comment:
Weird question here, but do we have a scenario where the `isUnauthorized` will be used not in conjunction with the platform check? Should we move this check closer to that provider?

Reply:
plus one to seeing if we can hoist it higher up the control flow, but if thats too much of a refactor what do you think about colocating the logic? i.e.

```typescript
const isUnauthorized = isPlatformiOS() && status === ENStatus.UNAUTHORIZED_DISABLED
```

Reply:
Good idea, tagged you both in a comment on a new commit.